ui-tests/cypress/integration/content/assets.spec.js

```diff
@@ -97,23 +109,19 @@ describe('Assets', () => {
           .then(response => content.id = response.body.payload[0].id);
         cy.contentsController().then(controller => controller.updateStatus(content.id, 'published'));
 
-        currentPage = openAssetsPage();
-        //TODO validate this wait
-        cy.wait(2000);
-
-        cy.get('@assetToBeDeleted').then(assetToBeDeleted =>
-            currentPage.getContent().getKebabMenu(assetToBeDeleted.id).open().clickDelete()
-        );
-        cy.wait(1000);
-        currentPage.getDialog().confirm();
-
-        cy.get('@assetToBeDeleted').then(assetToBeDeleted =>
-            cy.validateToast(currentPage, assetToBeDeleted.id, false)
-        );
-
-        cy.contentsController().then(controller => {
-          controller.updateStatus(content.id, 'draft');
-          controller.deleteContent(content.id);
+        openAssetsPage().then(page => {
+          cy.get('@assetToBeDeleted')
+            .then(() =>
+                page.getContent().getKebabMenu().openDropdown().clickDelete(isForbidden)
```

Comment:
??

Reply:
we have two different tates on clickDelete in this section:
one that redirects us to the DeleteAdminPage, and one that redirects us to a specific page when an asset cannot be deleted.
I inserted a boolean 'isForbidden' to be able to write a single method with an if-else inside that would check this addressing.